generators/app/index.js

```diff
@@ -81,38 +104,70 @@ module.exports = yeoman.generators.Base.extend({
         default: "Firstname Lastname",
         store: true
       },
+      {
+        type: 'input',
+        name: 'authorEmail',
+        message: 'Your email?',
+        store: true
+      },
+      {
+        type: 'input',
+        name: 'authorUrl',
+        message: 'Your home page url?',
+        store: true
+      }
 
     ];
 
     this.prompt(prompts, function (props) {
       this.props = props;
-      // To access props later use this.props.someOption;
       this.moduleName = props.moduleName;
       this.moduleDescription = props.moduleDescription;
+      this.hook = props.hook;
+      this.hookCallback = props.hookCallback;
+      if(this.hook == 'postEntity'){
+        this.hookType = 'post';
+        this.hookFor = 'entity';
+      }
       this.githubName = props.githubName;
       this.authorName = props.authorName;
+      this.authorEmail = props.authorEmail;
+      this.authorUrl = props.authorUrl;
       done();
     }.bind(this));
   },
 
-  writing: function () {
-    var done = this.async();
+  writing: {
+    writeCommonTemplates : function () {
+      this.copy('editorconfig', '.editorconfig');
+      this.copy('eslintrc', '.eslintrc');
+      this.copy('gitattributes', '.gitattributes');
+      this.copy('gitignore', '.gitignore');
+      this.copy('_travis.yml', '.travis.yml');
+      this.copy('_gulpfile.js', '.gulpfile.js');
```

Comment:
Ooops yes :)
On 23 Jan 2016 03:44, "Pascal Grimaud" <notifications@github.com> wrote:

> In generators/app/index.js
> <https://github.com/jhipster/generator-jhipster-module/pull/8#discussion_r50581441>
> :
>
> >        done();
> >      }.bind(this));
> >    },
> >
> > -  writing: function () {
> > -    var done = this.async();
> > +  writing: {
> > +    writeCommonTemplates : function () {
> > +      this.copy('editorconfig', '.editorconfig');
> > +      this.copy('eslintrc', '.eslintrc');
> > +      this.copy('gitattributes', '.gitattributes');
> > +      this.copy('gitignore', '.gitignore');
> > +      this.copy('_travis.yml', '.travis.yml');
> > +      this.copy('_gulpfile.js', '.gulpfile.js');
>
> should it be gulpfile.js instead of .gulpfile.js ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jhipster/generator-jhipster-module/pull/8/files#r50581441>
> .
>